### PR TITLE
AppliedInventories operation

### PR DESCRIPTION
**Pull request by slemrmartin**
- [x] **depends on** https://github.com/ManageIQ/topological_inventory-api-client-ruby/pull/22

**dependent** https://github.com/ManageIQ/topological_inventory-api/pull/257

### CODE REVIEW

**Comment by agrare** [comment]
@slemrmartin the name ApprovalInventories makes some assumptions about how this will be used (e.g. by catalog for the purpose of approvals).

Would something like ServiceOfferingInventories make sense?

**Comment by slemrmartin** [comment]
@agrare yes and no :) there shoud be `ServiceOffering.service_offering_inventories` method, and `/service_offering/:id/service_offering_inventories` URL in API - but it invokes that it's simple `has_many` list.
I decided for `approval_inventories`, because this operation is really Approval Service specific, I doubt it'll make sense for any other service.
It won't be used by catalog, it should provide different list for user input.

Maybe I can switch back to `effective_inventories`, but it does mean nothing.

**Comment by slemrmartin** [comment]
@agrare we can name it "applied_inventories". It describes the operation and is not connected to specific svc, what do you think?

**Comment by agrare** [comment]
@slemrmartin either applied or effective inventories sounds good to me :+1:

File: lib/topological_inventory/ansible_tower/operations/service_offering.rb [hunk 1]
```diff
@@ -2,7 +2,8 @@
 require "topological_inventory/ansible_tower/operations/core/ansible_tower_client"
 require "topological_inventory/ansible_tower/operations/core/service_order_mixin"
 require "topological_inventory/ansible_tower/operations/core/topology_api_client"
-
+require "topological_inventory/ansible_tower/operations/applied_inventories/parser"
+require "pry-byebug"
```
Comment (agrare) [inline lib/topological_inventory/ansible_tower/operations/service_offering.rb]: Don't forget to remove this

**Review by agrare — APPROVED** [review]
LGTM, take it out of wip when you're ready for this to get merged

File: lib/topological_inventory/ansible_tower/operations/service_offering.rb (hunk 1 above)
Comment (slemrmartin) [reply lib/topological_inventory/ansible_tower/operations/service_offering.rb]: I'm testing and writing specs now, but..yes, thanks, it happens to me :)

**Comment by slemrmartin** [comment]
@agrare it's ready now

**Pull request merged**